docs/application/dotnet/getting-started/features.md

```diff
@@ -0,0 +1,47 @@
+<a id="features"></a>
+# Tizen .NET Features and Components
+
+Tizen .NET enables you to build .NET applications with Xamarin.Forms and the Tizen .NET framework. Xamarin.Forms allows you to easily create a user interface, and the TizenFX API provides numerous interfaces to much of the hardware that is found in modern TV, mobile, wearable, and IoT devices.
+
+**Figure: Tizen .NET architecture**
+
+![Tizen .NET architecture](media/cs_overview.png)
+
+Tizen .NET consists of the following main components:
+
+- **.NET Core**
+
+  .NET Core is a general-purpose development platform maintained by Microsoft and the .NET community on [GitHub](https://github.com/dotnet/core). It is cross-platform, supporting Windows&reg;, macOS, and Linux, and can be used in device, cloud, and embedded/IoT scenarios.
+
+  .NET Core is composed of the following parts:
```

Comment:
.NET Core consists of the following parts:

Reply:
done